src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+@use "../utils/variables" as *;
+
+.card {
+  width: 200px;
+  padding: 33px 17px 17px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  background-image: url(../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  background-repeat: no-repeat;
+  background-size: contain;
+  align-self: center;
+}
+
+.card__nameProduct {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin-top: 40px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__codeProduct {
+  margin: 4px 0;
+  font-family: Roboto, sans-serif;
```

Comment:
make this font global to not repeating it so often

Reply:
font global maded